### DISCUSSION TITLE
Questions about alpha estimate

### DISCUSSION THREAD

**Discussion by huizizhang949** [theislab/scvelo#758] [Q&A] [answered]
Hi,

I am new to this package and learning it from the tutorial here https://scvelo.readthedocs.io/DynamicalModeling/. I have some questions about the results in this tutorial.

In code chunk 9, I wonder why the the estimated alpha parameter is a single value? From the paper of Bergen et al. (2020), I think there should be at least two states (induction and repression) with two different alpha, or four states including active/inactive stead states.

Besides, I am trying to do some simulations but not sure what values are sensible for the alpha vector, beta and gamma (same notations in the paper). Can you please give some suggestions?

Thank you!

Bergen, V., Lange, M., Peidli, S., Wolf, F.A. and Theis, F.J., 2020. Generalizing RNA velocity to transient cell states through dynamical modeling. Nature biotechnology, 38(12), pp.1408-1414.

**Comment by WeilerP** [comment, marked as answer]
@Huizi949-z, the transcription rate during repression is 0, and in steady states either equal to the one during induction or repression. The inferred transcription rate is the one for the induction phase.

**Reply by huizizhang949** [reply]
Thanks for the reply.!

'the transcription rate during repression is 0, and in steady states either equal to the one during induction or repression'
 Is this a general assumption for all cases or is it just specific to this example dataset? Because just from the paper, it seems that there could be at most four different alpha for each of the four states.

**Reply by WeilerP** [reply]
Well, you do have two different values of transcription rate: One for induction and one for repression. We know that transcription is turned off during repression, _i.e._, the rate is 0 and we do not need to estimate it. Since the steady states are either part of induction or repression and we assume constant rates during these steps, we estimate one set of parameters per phase.

**Reply by huizizhang949** [reply]
OK. I see. Thank you!
 
Besides, I woud like to simulate some datasets of unspliced and spliced counts following the dynamic model equations (4) in the Methods section. Could you please suggest some plausible values/ranges for alpha, beta and gamma?

**Reply by WeilerP** [reply]
This highly depends on what you would like to achieve, _i.e._, the shape of the phase portrait, scale, etc. Please see [here](https://github.com/theislab/scvelo_notebooks/blob/master/vignettes/FigSuppl.ipynb) for an exemplary simulated dataset.

**Reply by huizizhang949** [reply]
Thanks for the link!
Sorry for a follow-up question, what do you mean by 'active steady state'? Is the the period after the induction or before the induction?
